tracepath.c

```diff
@@ -384,6 +387,12 @@ static int probe_ttl(struct run_state *const ctl)
 
 	if (i < MAX_PROBES) {
 		data_wait(ctl);
+		const int allow_max =
+			(ctl->ai->ai_family==AF_INET6)?(DEFAULT_MTU_IPV6):(DEFAULT_MTU_IPV4);
```

Comment:
nit: Brackets aren't needed. I would also use the constants directly
```
if (ctl->mtu > (ctl->ai->ai_family==AF_INET6 ? DEFAULT_MTU_IPV6 : DEFAULT_MTU_IPV4)
    error(2, 0, "MTU <= MAX_MTU");
```
Return isn't needed.